test/apicoverage.sh

```diff
@@ -52,7 +52,7 @@ ko apply -f "${TMP_DIR}/test/apicoverage/image/apicoverage-webhook.yaml" || fail
 header "Running tests"
 # Run conformance tests and e2e tests
 go_test_e2e -timeout=30m ./test/conformance/api/v1alpha1 ./test/conformance/api/v1beta1 ./test/conformance/runtime ./test/e2e \
-  --systemNamespace=${E2E_SYSTEM_NAMESPACE} || fail_apicoverage_run "Failed in executing Tests"
```

Comment:
If we run this without setting `SYSTEM_NAMESPACE` do things still work?

Reply:
Good point, will add a default.